src/shilpa/Assignment_32/SwapStrWithoutTempVariable.java

```diff
@@ -0,0 +1,23 @@
+/*program 2: swap 2 String without using temporary variable.
+input : input1 = "Techno" , input2 = "Credits";
+output : input1 = "Credits", input2 = "Techno"
+Hint : you can use subString method.
+*/
+package shilpa.Assignment_32;
+
+public class SwapStrWithoutTempVariable {
+	void getSwapString(String str1, String str2) {
+		str1 = str1 + str2;
+		str2 = str1.substring(0, 6);
```

Comment:
This should not be hard coded

Reply:
https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/1541/files

take reference